apps/portal/src/content/docs/components/button-options.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: ButtonWithOptions
+description: ButtonWithOptions component documentation
+---
+
+The `ButtonWithOptions` component provides a flexible UI element for displaying a button with a dropdown of options. It can behave as a radio button group if a `selectedValue` is provided, or as a regular dropdown otherwise.
+
+import { DocsPage } from "@/components/docs-page";
+import { ButtonWithOptions } from "@harnessio/ui/components";
+import { Button } from "@harnessio/ui/components";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<div className="space-y-4">
+    <div className="mb-4 text-white">
```

Comment:
Try to avoid coloring the text with specific colors, we'll be adding the theme selector to this app and this will disappear in `light` mode.
```suggestion
    <div className="mb-4">
```